testdata.log

```diff
@@ -8412,23 +8412,17 @@ SELECT * FROM t WHERE  EXISTS ( SELECT * FROM t WHERE i == 2 ) ORDER BY i;
 
 ---- 1349
 SELECT * FROM t WHERE  EXISTS ( SELECT * FROM t WHERE i == 2 );
```

Comment:
> Is that okay?

The plan is a transformation of the SQL statement showing how the statement will be executed. The semantics of the statement shall be preserved, but the plan above does not do that. Unfortunately, I'm not sure what the correct plan rendering should look like right now, so I suggest to revisit this later. It is not a priority so long as the plan works correctly.